runtime/ai/developer_agent.go

```diff
@@ -19,8 +19,9 @@ type DeveloperAgent struct {
 var _ Tool[*DeveloperAgentArgs, *DeveloperAgentResult] = (*DeveloperAgent)(nil)
 
 type DeveloperAgentArgs struct {
-	Prompt      string `json:"prompt" jsonschema:"User prompt for the developer agent."`
-	InitProject bool   `json:"init_project,omitempty" jsonschema:"Set to true if the prompt is provided as part of project initialization. Should only be used when the project directory contains an empty project (except for the boilerplate generated by UnpackEmpty)."`
+	Prompt          string `json:"prompt" jsonschema:"User prompt for the developer agent."`
+	InitProject     bool   `json:"init_project,omitempty" jsonschema:"Set to true if the prompt is provided as part of project initialization. Should only be used when the project directory contains an empty project (except for the boilerplate generated by UnpackEmpty)."`
+	CurrentFilePath string `json:"current_file_path,omitempty" jsonschema:"Optional path to the file that the user is currently viewing/editing."`
```

Comment:
This needs to be manually passed through here also: https://github.com/rilldata/rill/blob/e03c48f9466e0a35f2b026a9dd25f8fe5f251a44/runtime/server/chat.go#L288-L290